screens/joinLeague.js

```diff
@@ -21,9 +24,15 @@ export default class JoinLeague extends Component {
    }
 
    submitJoinLeague = () => {
-      joinLeague(this.state.inviteCode, this.state.nickname).then((res) => {
-         // TODO: Set current portfolio in store to joined portfolio
-         Actions.portfolioMain();
+      joinLeague(this.state.inviteCode, this.state.nickname).then(async (joinRes) => {
```

Comment:
Can you continue to follow the async await pattern here?

Reply:
Yeah good call!